src/components/presentational/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,41 @@
+import "./ProgressBar.css";
+import React, { ReactElement, useEffect } from "react";
+
+export interface ProgressBarProps {
+	background?: string;
+	fill?: string;
+	fillPercent: number;
```

Comment:
I think we can add this component to the view builder just by itself, but that requires a couple additional props.

We need to add the "innerRef" prop so this can be dragged/dropped, and I also would support a "defaultMargin" prop that we would set to true for the view builder which would add the default 16px margin to this (see the link for example)